tardis/io/util.py

```diff
@@ -265,7 +269,51 @@ def to_hdf(self, file_path, path='', name=None):
         data = self.get_properties()
         buff_path = os.path.join(path, name)
         self.to_hdf_util(file_path, buff_path, data)
-        
+
+
+class PlasmaWriterMixin(HDFWriterMixin):
+
+    def get_properties(self, collection=None):
+        try:
+            data = {name: getattr(self, name) for name in self.outputs}
+        except AttributeError:
+            if collection:
+                data = {name.__class__.__name__: name for name in self.plasma_properties
+                        if isinstance(name, tuple(collection))}
+            else:
+                data = {name.__class__.__name__: name for name in self.plasma_properties}
+                data['atom_data_uuid'] = self.atomic_data.uuid1
+            if 'atomic_data' in data:
+                data.pop('atomic_data')
+        return data
+
+    def to_hdf(self, file_path, path='plasma', collection=None):
```

Comment:
@wkerzendorf I am facing a small problem here , I don`t want `name` parameter here for `Plasma hdf calls` , 
With `name` parameter it will save like ex-> `/plasma/abundance/abundance`
without name parameter `/plasma/abundance/`